backend/oasst_backend/api/v1/messages.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Query, Response
+from oasst_backend.api import deps
+from oasst_backend.api.v1 import utils
+from oasst_backend.exceptions import OasstError, OasstErrorCode
+from oasst_backend.models import ApiClient
+from oasst_backend.models.db_payload import PostPayload
+from oasst_backend.prompt_repository import PromptRepository
+from oasst_shared.schemas import protocol
+from sqlmodel import Session
+from starlette.status import HTTP_200_OK
+
+router = APIRouter()
+
+
+@router.get("/")
+def query_messages(
+    username: str = None,
+    api_client_id: str = None,
+    max_count: int = Query(10, gt=0, le=25),
```

Comment:
I think we can safely query/send a bit more data, e.g. `le=1000`.

Reply:
+